src/templates/FilteredResource/QueryRow.jsx

```diff
@@ -29,8 +29,13 @@ const QueryRow = ({ id, condition, index, update, remove, propertyOptions, schem
       } else {
         update(index, 'property', '');
       }
+      if (schema[id].fields[condition.property].mysql_type === 'date') {
+        if(!value) {
+          setValue(startDate.toJSON().slice(0, 10));
```

Comment:
This would be clearer if the number 10 was assigned to a constant -- I don't understand what this parsing accomplishes.

Reply:
To match the value of a date field coming out of the DKAN datastore we have to match `YYYY-MM-DD`. This is the JSON format, but JavaScript adds all the the `:HH:MM`... to it so the slice is there to get the the first part. It will never change that I know of which is why I didn't set it as a variable. When data dictionaries are finally in DKAN we might need to revisit if there are different date types. Right now this only exists for a specific set of datasets on Data.Medicaid.